src/main/java/Duke.java

```diff
@@ -1,10 +1,76 @@
+import java.util.Objects;
+import java.util.Scanner;
+
 public class Duke {
+    public static String concatenateChar(String line, String start, String end) {
+        int indexOfStart = line.indexOf(start) + 1;
+        int indexOfEnd = 0;
+        if (end == null) {
+            indexOfEnd = line.length();
+        } else {
+            indexOfEnd = line.indexOf(end);
+        }
+        String taskName = line.substring(indexOfStart, indexOfEnd);
+        return taskName;
+    }
+
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
+        String logo = " _           _        \n"
+                + "| |    _   _| | _____ \n"
+                + "| |   | | | | |/ / _ \\\n"
+                + "| |_  | |_| |   <  __/\n"
                 + "|____/ \\__,_|_|\\_\\___|\n";
         System.out.println("Hello from\n" + logo);
+        System.out.println("Hello! I'm Luke");
+        System.out.println("What can I do for you?");
+        Task[] taskList = new Task[100];
+        Scanner in = new Scanner(System.in);
+        String line = in.nextLine();
+
+        while (!line.equals("bye")) {
+            if (line.equals("list")) {
+                System.out.println("Here are the tasks in your list:");
+                for (int taskNo = 0; taskNo < Task.getNoOfTasks(); taskNo++) {
+                    System.out.println((taskNo + 1) + ". " + taskList[taskNo]);
+                }
+            }
+
+            if (line.contains("todo")) {
+                String toDoName = concatenateChar(line, " ", null);
+                taskList[Task.getNoOfTasks()] = new ToDos(toDoName);
+                taskList[Task.getNoOfTasks() - 1].taskIcon = "T";
+                System.out.println("Got it! I have added this task:");
+                System.out.println(taskList[Task.getNoOfTasks() - 1]);
+                System.out.println("Now you have " + Task.getNoOfTasks() + " tasks in your list");
```

Comment:
Is it possible to extract method for the print lines here? This can potentially make the code neater and easier to read.